### ISSUE TITLE
UPnP: this module was disabled at compile-time

### ISSUE THREAD

**Issue by dannykorpan** [diva-exchange/i2p#4]
Hi,

I've got the following error when using ENABLE_UPNP=1 as environment variable with the latest divax/i2p:current-i2p-tor docker image.
```
[01/Jan/2024:16:23:22 +0000]@579/warn - UPnP: this module was disabled at compile-time
```

Kind Regards,
Danny

**Comment by diva-exchange** [comment]
Confirmed, thanks. The build is without UPnP: 

Bug, within dockerfile -> https://github.com/diva-exchange/i2p/blob/a5e64c61cb66571067d7532f7484144ed3e97598/Dockerfile#L40

instead of:
`&& cmake -DWITH_AESNI=ON -DWITH_UPNP=ON . \`


Two possible solutions:
a) remove the ENABLE_UPNP config option
b) set UPNP compile flag to ON.

**Comment by diva-exchange** [comment]
-> Set UPNP compile flag to ON.
Fixed in Release v2.50.2-2024-01-20: https://github.com/diva-exchange/i2p/releases/tag/v2.50.2-2024-01-20
